cypress/e2e/shared/dashboardsView.test.ts

```diff
@@ -1392,4 +1392,198 @@ csv.from(csv: data) |> filter(fn: (r) => r.bucket == v.bucketsCSV)`
+          cy.visit(`${orgs}/${orgID}/dashboards/${body.id}`)
+          cy.getByTestID('tree-nav')
+        })
+      })
+      cy.window().then(win => {
+        win.influx.set('pauseCell', true)
+        win.influx.set('newAutoRefresh', true)
+      })
+
+      cy.createBucket(orgID, name, 'schmucket')
+
+      const now = Date.now()
+      cy.writeData(
+        [
+          `test,container_name=cool dopeness=12 ${now - 1000}000000`,
+          `test,container_name=beans dopeness=18 ${now - 1200}000000`,
+          `test,container_name=cool dopeness=14 ${now - 1400}000000`,
+          `test,container_name=beans dopeness=10 ${now - 1600}000000`,
+        ],
+        'schmucket'
+      )
+    })
+
+    cy.getByTestID('button').click()
+    cy.getByTestID('switch-to-script-editor').should('be.visible')
+    cy.getByTestID('switch-to-script-editor').click()
+    cy.getByTestID('toolbar-tab').click()
+
+    const query1 = `from(bucket: "schmucket")
+|> range(start: v.timeRangeStart, stop: v.timeRangeStop)
+|> filter(fn: (r) => r["container_name"] == "cool")`
+
+    const query2 = `from(bucket: "schmucket")
+|> range(start: v.timeRangeStart, stop: v.timeRangeStop)
+|> filter(fn: (r) => r["container_name"] == "beans")`
+
+    cy.getByTestID('flux-editor')
+      .should('be.visible')
+      .click()
+      .focused()
+      .type(query1)
+
+    cy.getByTestID('overlay').within(() => {
+      cy.getByTestID('page-title').click()
+      cy.getByTestID('renamable-page-title--input')
+        .clear()
+        .type('blah')
+      cy.getByTestID('save-cell--button').click()
+    })
+
+    cy.getByTestID('button').click()
+    cy.getByTestID('switch-to-script-editor').should('be.visible')
+    cy.getByTestID('switch-to-script-editor').click()
+    cy.getByTestID('toolbar-tab').click()
+
+    cy.getByTestID('overlay').within(() => {
+      cy.getByTestID('flux-editor')
+        .should('be.visible')
+        .click()
+        .focused()
+        .type(query2)
+      cy.getByTestID('save-cell--button').click()
+    })
+
+    cy.intercept('POST', 'query', req => {
+      if (req.body.query === query1) {
+        done()
+      }
+      if (req.body.query === query2) {
+        req.alias = 'secondCellQuery'
+      }
+    })
+    cy.getByTestID('cell blah').within(() => {
+      cy.getByTestID('cell-context--toggle').click()
+    })
+    cy.getByTestID('cell-context--pause').click()
+
+    cy.getByTestID('enable-auto-refresh-button').click()
+    cy.getByTestID('auto-refresh-overlay').should('be.visible')
+    cy.getByTestID('auto-refresh-overlay').within(() => {
+      cy.getByTestID('autorefresh-dropdown--button').click()
+      cy.getByTestID('auto-refresh-5s').click()
+      cy.getByTestID('refresh-form-activate-button').click()
+      cy.wait('@secondCellQuery')
+    })
+    cy.getByTestID('cell blah').within(() => {
+      cy.getByTestID('cell-context--toggle').click()
+    })
+    cy.getByTestID('cell-context--pause').click()
```

Comment:
How is this test testing the pause functionality?

Reply:
Because the `done` call in the first intercept if clause will only fire if the first query is fired, which will only happen once it is unpaused, so this is testing that after we unpause, it goes back inside the autorefresh loop

Reply:
Gotcha. That makes sense. Is there any way we could get a comment on the test to explain what's happening because I know I'm gonna see this again and be like "what?"

Reply:
Done. Pushed up a comment to explain it